src/assets/styles/layout.scss

```diff
@@ -109,9 +109,29 @@ $breakpoints: (
   @include respond-above(sm) {
     display: grid;
     grid-template-columns: 1fr minmax(auto, $containerFormMaxWidth) 1fr;
-    
+
     > * {
         grid-column: 2;
     }
+  }
+}
+
+@mixin heroContainer() {
```

Comment:
So the resulting media queries will be targeted at (max-width 1199px), (min-width 992px) which doesn't make much sense unless I'm missing something here. Wouldn't it be better to write this in a mobile-first manner?
```
@mixin heroContainer() {
  display: grid;
  grid-template-columns: 1fr 10fr 1fr;
  > * {
    grid-column: 2;
  }
  @include respond-above(md) {
    grid-template-columns: 1fr minmax(auto, $containerFormMaxWidth) 1fr;
  }
}
```